src/options.rs

```diff
@@ -119,8 +119,10 @@ pub struct BuildOptions {
     /// Dead code is stripped by default.
     /// This flag allows you to opt out and always include dead code.
     /// Please note, this could trigger unexpected behavior or even ICEs in the compiler.
-    #[arg(long, default_value_t = true)]
-    pub strip_dead_code: bool,
+    //
+    // `Option<Option<bool>>` gives an optional argument with an optional value.
+    #[arg(long)]
+    pub strip_dead_code: Option<Option<bool>>,
```

Comment:
Even though testing the actual behavior of whether dead code is included in a final artifact is a bit tricky and overkill, mind testing hat we can at least pass all of

* `--strip-dead-code`
* `--strip-dead-code=true`
* `--strip-dead-code=false`

through the CLI parser?

Cribbing from a simple test like https://github.com/rust-fuzz/cargo-fuzz/blob/97b64226cccbfc64d7491f966b8737e609146229/tests/tests/main.rs#L258-L312 and wrapping that up with a loop like

```rust
for flag in ["--strip-dead-code", "--strip-dead-code=true", "--strip-dead-code=false"] {
    // ...
}
```

should do the trick.